.github/workflows/python-ci-wheel.yml

```
@@ -82,15 +82,14 @@ jobs:
       #===============================================#
       # wheels
       - name: Build wheels
-        uses: joerick/cibuildwheel@f3dae81e82da2a2a36338269e32f3adb7b42dc8c
+        uses: pypa/cibuildwheel@v2.3.1
         with:
           output-dir: wheelhouse
         env:
           CIBW_SKIP: cp27-* # manylinux2014 not compatible with python 2.7
           CIBW_BUILD: ${{ env.CIBW_BUILD_IDENTIFIER }}
-          # TODO: change back to "manylinux2014" when cibuildwheel releases new version
-          CIBW_MANYLINUX_X86_64_IMAGE: quay.io/pypa/manylinux2014_x86_64:2021-02-17-8d1372a
-          CIBW_MANYLINUX_I686_IMAGE: quay.io/pypa/manylinux2014_i686:2021-02-17-8d1372a
+          CIBW_MANYLINUX_X86_64_IMAGE: quay.io/pypa/manylinux2014_x86_64
+          CIBW_MANYLINUX_I686_IMAGE: quay.io/pypa/manylinux2014_i686
```

```suggestion
          CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
          CIBW_MANYLINUX_I686_IMAGE: manylinux2014
```

Cibuildwheel seems to default to manylinux2014 now, so it should be safe to write it like this or even remove these two lines completely.